lib/puppet-syntax/manifests.rb

```diff
@@ -36,6 +36,11 @@ def check(filelist)
         e =~ /^You cannot collect( exported resources)? without storeconfigs being set/
       }
 
+      # tag parameter in class raise warning
```

Comment:
Can the comment here be amended to reflect the comment above? This one's a bit less descriptive than the one above!